com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Accumulation/SubFrameManager.cs

```diff
@@ -67,13 +112,14 @@ void Init(int samples, float shutterInterval)
             m_ShutterInterval = samples > 1 ? shutterInterval : 0;
             m_IsRecording = true;
             m_IsRenderingTheFirstFrame = true;
-            Reset();
+
+            Clear();
 
             m_OriginalTimeScale = Time.timeScale;
 
             Time.timeScale = m_OriginalTimeScale * m_ShutterInterval / m_AccumulationSamples;
 
-            if (m_Centered && m_IsRenderingTheFirstFrame)
```

Comment:
Perhaps I should have put a comment in the code, but we do need the check for m_IsRenderingTheFirstFrame.

The idea is that for centered motion blur, we "squeeze" the duration of the first frame to half (with timescale *=0.5), so the rest of the frames are correctly centered. But this extra scaling needs to be done only for the first frame.

Reply:
(I only removed it in the Init(), not in the PrepareNewSubFrame())

Reply:
Oh yeah, you are right, I though this was PrepareNewSubFrame, which has the same logic.